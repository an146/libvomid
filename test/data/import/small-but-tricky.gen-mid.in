#!/bin/sh

source gen-mid.sh

header 2
track <<EOT
	pitchwheel 1 0x2000
	ctrl 10 7 100
	on 1 10 20
	delay 200
	pitchwheel 1 0x2001
	pitchwheel 1 0x1FFF
	ctrl 11 7 101
	off 1 10 20
	pitch 11
	on 1 10 30
	delay 100
	off 2 10 15
	off 2 10 66
	on 2 10 20
	delay 128
	end
EOT
track <<EOT
	notesystem 3
	delay 250
	pitch 16383
	on 2 10 5
	delay 50
	program 9 0
	program 9 1
	program 9 0
	notesystem 4
	off 1 10 35
	pitch 16384
	on 1 10 10
	delay 10
	program 9 123
	pitchwheel 1 0x2001
	meta 01 "xyi"
EOT
