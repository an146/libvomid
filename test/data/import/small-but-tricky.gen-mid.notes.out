0 200 20 20 10 10 1|200 300 30 35 11 10 1|300 428 20 64 10 10 2|
250 300 5 66 16383 10 2|300 310 10 64 16384 10 1|
